_ide_helper_models.php

```diff
@@ -1928,17 +1928,17 @@ class ProjectRole extends \Eloquent {}
  * @property-read \App\Models\Custodian $custodian
  * @property-read \App\Models\Project $project
  * @property-read \App\Models\User $user
- * @method static \Illuminate\Database\Eloquent\Builder<static>|ProjectUserCustodianApproval newModelQuery()
```

Comment:
I think this file should not be sent to GitHub.

Reply:
wasnt added by me - but you may be correct that isnt needed in github